examples/c8000v-ha/main.tf

```diff
@@ -0,0 +1,71 @@
+provider "equinix" {
+  client_id     = var.equinix_client_id
+  client_secret = var.equinix_client_secret
+}
+
+module "c8000v_ha" {
+  source               = "../../modules/c8000v"
+  name                 = "tf-c8000v-router-ha"
+  metro_code           = var.metro_code_primary
+  platform             = "small"
+  byol                 = true
+  account_number       = "664566"
+  license_token        = ""
```

Comment:
If license token is optional, we should ignore this field in the example or set up a default value if this is not passed by customer.